core/src/main/java/org/infinispan/configuration/cache/L1Configuration.java

```diff
@@ -46,18 +49,37 @@ public boolean enabled() {
     * </p>
     *
     * <p>
-    * By default multicast will be used.
+    * If the threshold is set to a negative number, then unicasts will always be used. If the threshold is set to a
+    * positive number, then multicast will be used if the number of nodes to send to exceeds this number.
     * </p>
     *
     * <p>
-    * If the threshold is set to -1, then unicasts will always be used. If the threshold is set to 0, then multicast
-    * will be always be used.
+    * By default this is not used and {@link #invalidationThresholdRatio()} will be used instead.
```

Comment:
This part here in the doc doesn't seem to follow what the code is doing. I think the phrasing would mention this is utilized along with `invalidationThresholdRatio` value.